resources/assets/less/bem/osu-page-header.less

```diff
@@ -156,6 +156,16 @@
     .@{top}:hover & {
       opacity: 1;
     }
+
+    & .forum-post-actions {
```

Comment:
this should be defined in `forum-post-actions.less` instead and scoped to just the wiki